create_tables.rb

```diff
@@ -10,6 +10,10 @@ def normalize_types(type, sql_type)
   elsif %w(uuid).include?(sql_type)
     # https://gist.github.com/wrobstory/4b0ce4e8ba51ec40c494881bc126c003
     'CHAR(36)'
+  elsif %w(datetime).include?(type)
+    # Redshift timestamp column does not support precision, but some of our newer timestamp columns include it,
+    # so normalize on the non-precision version
+    'TIMESTAMP WITHOUT TIME ZONE'
```

Comment:
Is the default more or less precise than what we're using? Any chances we run into problems later when we import the rows in Redshift?

Reply:
According to [Redshift docs](https://docs.aws.amazon.com/redshift/latest/dg/r_Datetime_types.html#r_Datetime_types-timestamp), "TIMESTAMP columns store values with up to a maximum of six digits of precision for fractional seconds". If that's what the `(6)` means in our newer tables, then that's probably actually the same?

In my staging testing, the 3 tables created with this updated DDL were able to load data fine.